tests/queries/0_stateless/03094_recursive_type_proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+SCHEMADIR=$CURDIR/format_schemas
```

Comment:
It appeared to be wrong, but shellcheck was broken for a bit https://s3.amazonaws.com/clickhouse-test-reports/0/51ddbf7099e4a79d9a6057391f44da62cdb0352e/style_check/shellcheck.txt

Please, check if the fix is correct in https://github.com/ClickHouse/ClickHouse/pull/62761